### ISSUE TITLE
Remove or refactor duplicate content in HTML artices in Learn area

### ISSUE THREAD

**Issue by dipikabh** [mdn/content#33046]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics#anatomy_of_an_html_element

### What information was incorrect, unhelpful, or incomplete?

This issue has been raised in https://github.com/orgs/mdn/discussions/658. 

Some of the sections and content across these two pages is duplicate or very similar:
https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started

For example:
- https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics#anatomy_of_an_html_element and
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#anatomy_of_an_html_element
- https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics#nesting_elements and
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#nesting_elements

### What did you expect to see?

Content that is not dupliacte

**Comment by awxiaoxian2020** [comment]
Any experts want to put forward for this? ping politely @bsmth and @wbamberg to get some good ideas :)

**Comment by wbamberg** [comment]
Sorry to be slow getting to this, @awxiaoxian2020 , starting to look at it now :).

**Comment by wbamberg** [comment]
I'm not sure what's best here, and wonder what @chrisdavidmills thinks. Yes, a fair amount of https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics#so_what_is_html is duplicated from https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started, although there are more or less editorial divergences.

And yes, duplication in general is not great. But these pages serve slightly different audiences.

- "Getting started with the web" is a crash course in the absolute basics of front-end, including HTML, CSS, JS, and tooling.
- "Getting started with HTML" is the first chapter in a more elaborate and advanced HTML course.

Both chapters need to talk about, for instance, the anatomy of an HTML element, but they don't at all share the same flow, so trying to de-dupe the content is likely to make the content less useful for people following one or the other course.

So I'm inclined to think that the duplication is the best option at the moment, and we should close this issue..

**Comment by awxiaoxian2020** [comment]
I agree that they don't at all share the same flow. if we de-dupe the content, we need more efforts. But I think we should design a new pattern or mode or structure for these content. It just give readers a good experience :)

**Comment by Josh-Cena** [comment]
I also agree that very little can be done. Realistically speaking it's a small duplication. And it just serves as a natural segue into talking about other things like attributes and elements. If you replace either occurrence with "See the other page for an anatomy of the element", you will just make everything else appear out of context.

**Comment by awxiaoxian2020** [comment]
My vision is to completely restructure these contents rather than just doing some simple removals to provide beginners with a good experience. The beginners around me are all confused about why the previously introduced content appears again. At the very least, we need an explanation to introduce the relationships between these contents.

**Comment by bsmth** [comment]
Thanks all. We can probably revisit this now that the Learn moving has been completed. @chrisdavidmills do you agree that duplication is intended and okay here? Anything that stands out that we could or should mention in terms of duplication of content?

**Comment by chrisdavidmills** [comment]
For a beginners' course, a bit of repetition is not necessarily a bad thing. When you meet a concept multiple times, it helps it to stick in your memory.

Now that I've done the big Learn restructure, the duplication is perhaps a bit more obvious.

Previously, the material was not in the same pathway, so there was a greater chance that you'd encounter https://developer.mozilla.org/en-US/docs/Learn_web_development/Getting_started/Your_first_website/Creating_the_content as a complete beginner's HTML "taster", but not necessarily dive into the more substantial HTML course starting at https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Structuring_content/Basic_HTML_syntax.

Now, however, they are explicitly all the same pathway, and you will definitely come across both if you continue to follow it.

To fix this, I would like to suggest that we edit https://developer.mozilla.org/en-US/docs/Learn_web_development/Getting_started/Your_first_website/Creating_the_content to replace the directly copied content with a more abbreviated version of the HTML anatomy information - a quick summary rather than a deep guide. The reader doesn't depth at this point.

What do we think?

**Comment by bsmth** [comment]
Thanks a lot!

> replace the directly copied content with a more abbreviated version of the HTML anatomy information - a quick summary rather than a deep guide

I'm +1 on this, by the way. Curious what others think.

**Comment by awxiaoxian2020** [comment]
> Thanks a lot!
> 
> > replace the directly copied content with a more abbreviated version of the HTML anatomy information - a quick summary rather than a deep guide
> 
> I'm +1 on this, by the way. Curious what others think.

+1 here

**Comment by Josh-Cena** [comment]
Happy to explore better structuring but I don't think we should treat these sections as sequential.
